packages/torghostng/PKGBUILD

```
@@ -0,0 +1,36 @@
+pkgname=TorghostNG
```

Only use lower case for pkgname please

```suggestion
pkgname=torghostng
```